cli/lib/appdata.ts

```diff
@@ -114,8 +114,7 @@ export async function getAuthToken(): Promise< NonNullable< UserData[ 'authToken
 		return authToken;
 	} catch ( error ) {
 		throw new LoggerError(
-			__( 'Authentication required. Please run the Studio app and log in to WordPress.com first.' ),
-			error
+			__( 'Authentication required. Please run the Studio app and log in to WordPress.com first.' )
```

Comment:
This change is unrelated to the core purpose of this PR, but it's so small I didn't think it deserved its own PR. Not passing the previous error to `LoggerError` means that the CLI will print the following error message:

```
✖ Authentication required. Please run the Studio app and log in to WordPress.com first.
```

Previously, it would print:

```
✖ Authentication required. Please run the Studio app and log in to WordPress.com first.: Authentication required
```